src/bin/casc/args.rs

```diff
@@ -1,13 +1,25 @@
 use clap::Parser;
 
 #[derive(Parser, Debug)]
-#[clap(author, version, name = "casc", about = "Compile Cascade SELinux policies into CIL", long_about = None)]
+#[clap(
+    author,
+    version,
+    name = "casc",
+    about = "Compile Cascade SELinux policies into CIL",
+    long_about = "Compile Cascade SELinux policies into CIL
+Supported use cases:
+    - Combine all policies: This is the default. It outputs a monolithic system policy with the system configuration options set to default values. \"-o <out_filename>\" to specify the output filename.
+    - Build one or more defined systems: \"-s <system_names>\"
+    - Build all defined systems: \"-s all\""
+)]
 pub struct Args {
-    /// List of input files to process.  Directories are searched recursively.
+    /// List of input files to process. Directories are searched recursively.
     #[clap(required(true))]
     pub input_file: Vec<String>,
+    /// Combine all policies.
```

Comment:
This comment needs to specify that we build them with the name of the argument, something like "Combine all policies into one system and generate a cil file named OUT_FILENAME".